conf/routes

```diff
@@ -2,16 +2,22 @@
 # This file defines all application routes (Higher priority routes first)
 # ~~~~
 
-GET         /                     io.pathfinder.controllers.ViewController.index
+GET           /                     io.pathfinder.controllers.ViewController.index
```

Comment:
I mentioned it on the last PR, but no one fixed it: We should not have a viewcontroller since we should have no views.